tofu/data/_core.py

```diff
@@ -2026,7 +2026,11 @@ def _extract_common_params(obj0, obj1=None):
 
     @staticmethod
     def _recreatefromoperator(d0, other, opfunc):
-        if type(other) in [int,float,np.int64,np.float64]:
+        if type(other) in [int, float, np.int64, np.float64]:
+            data = opfunc(d0.data, other)
+            dcom = d0._extract_common_params(d0)
+
+        elif isinstance(other, np.ndarray):
             data = opfunc(d0.data, other)
```

Comment:
I would put an "or" in the if, instead of showing two different cases with the same result

Reply:
You(re right, but I wanted to separate these becasue I may to treat these cases differently later, let's say it for readability and anticipation of future needs